src/cpp/transport/tcp/RTCPMessageManager.cpp

```diff
@@ -579,7 +577,7 @@ bool RTCPMessageManager::processRTCPMessage(TCPChannelResource *pChannelResource
     {
     case BIND_CONNECTION_REQUEST:
     {
-        logInfo(RTCP_SEQ, "Receive [BIND_CONNECTION_REQUEST] Seq: " << controlHeader.transactionId);
+        //logInfo(RTCP_SEQ, "Receive [BIND_CONNECTION_REQUEST] Seq: " << controlHeader.transactionId);
```

Comment:
Why have you removed all the logs?

Reply:
RTCP_SEQ logs were added with debug purposes and in its current state can introduce some noise.